app/helpers/goldencobra/articles_helper.rb

```diff
@@ -46,22 +46,24 @@ def parse_glossar_entries(content,tag_name, parent_article_id=nil)
     # @param options={} [Hash] {link_image_size: :thumb, target_image_size: :large}
     #
     # @return [HTML] ImageGallery
-    def render_article_image_gallery(options={})
-      link_image_size = options.fetch(:link_image_size, :thumb)
-      target_image_size = options.fetch(:target_image_size, :large)
-      if @article
-        result = ""
-        uploads = Goldencobra::Upload.tagged_with(@article.image_gallery_tags.present? ? @article.image_gallery_tags.split(",") : "" )
-        if uploads && uploads.count > 0
-          uploads.order(:sorter_number).each do |upload|
-            result << content_tag("li", link_to(image_tag(upload.image.url(link_image_size), {alt: upload.alt_text}), upload.image.url(target_image_size), title: raw(upload.description)))
+    def render_article_image_gallery(options = {link_image_size: :thumb, target_image_size: :large})
+      if @article && @article.image_gallery_tags.present?
+        tags = @article.image_gallery_tags.split(",")
+        uploads = Goldencobra::Upload.tagged_with(tags)
+        list_items = ""
+        uploads.order(:sorter_number).each do |upload|
+          list_items << content_tag("li") do 
+            link_to upload.image.url(options[:target_image_size]), title: raw(upload.description) do
+              image_tag(upload.image.url(options[:link_image_size]), {alt: upload.alt_text})
```

Comment:
Line is too long. [92/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.